packages/client/hooks/useSpotlightSimulatedDrag.tsx

```diff
@@ -54,6 +54,10 @@ const useSpotlightSimulatedDrag = (
       const meetingProxy = store.get(meetingId)
       meetingProxy?.setValue(null, 'spotlightGroup')
       meetingProxy?.setValue(null, 'spotlightReflectionId')
+      meetingProxy?.setValue(null, 'spotlightSearchQuery')
+      const reflection = store.get(reflectionId)
+      // set isDropping to true so that the source is added back to its original position in kanban
+      reflection?.setValue(true, 'isDropping')
```

Comment:
i'm pretty sure this line was removed @nickoferrall can you confirm if we can remove this?

Reply:
Yeah I think it's removed. I overlooked it in conflict resolution.

Reply:
Yep, should be removed